### ISSUE TITLE
Add Privacy Manifest

### ISSUE THREAD

**Issue by hkellaway** [Iterable/swift-sdk#750]
Hello -

I'm wondering what plans there are to add a Privacy Manifest? Thanks!

https://developer.apple.com/support/third-party-SDK-requirements/
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api

**Comment by djs-code** [comment]
+1.  The App Store will be enforcing this more strictly starting on May 1st.

**Comment by Ayyanchira** [comment]
Thank you for your patience @hkellaway and @djs-code .
Next release will have privacy manifest included :)

**Comment by evantk91** [comment]
6.5.1 has been released with the privacy manifest

**Comment by djs-code** [comment]
@Ayyanchira @evantk91  I don't believe this is truly complete yet; if my understanding is correct, you will also need to add the following to the `IterableSDK` target in your `Package.swift`:

resources: [.process("PrivacyInfo.xcprivacy")]
